script/make_utils/check_pytest_determinism.sh

```diff
@@ -33,12 +33,35 @@ then
     exit 255
 fi
 
-set -e
-
 # Exceptions:
-#   passed in: since it is related to timings
-diff "${OUTPUT_DIRECTORY}/one.txt" "${OUTPUT_DIRECTORY}/two.txt" -I "passed in"
-echo "Successful determinism check"
+#   in X.Xs: since it is related to timings
+diff_output=$(diff "${OUTPUT_DIRECTORY}/one.txt" "${OUTPUT_DIRECTORY}/two.txt" -I "in [0-9]*\.[0-9]*s")
+
+# If a diff is present, we need to print the tests that failed to be reproduced for debugging.
+if [ -n "$diff_output" ]; then
```

Comment:
not sure I get everything here but I trust you !

Reply:
Yeah it wasn't as straightforward as I thought it would be. Basically, I print lines that are above the diffs for better debugging.